package.json

```diff
@@ -34,5 +34,20 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "big.js": "^6.0.3",
```

Comment:
`big.js` package should be added as a regular dependency instead of devDependencies. Packages added as devDependencies are only used during development and not into production.